apps/zotonic_mod_survey/src/models/m_survey.erl

```diff
@@ -205,6 +206,10 @@ m_get([ <<"result_columns">>, SurveyId, Format | Rest ], _Msg, Context) ->
         {error, _} = Error ->
             Error
     end;
+m_get([ <<"get_stored">>, UserId | Rest ], _Msg, Context) ->
+    {ok, {get_stored(m_rsc:rid(UserId, Context), Context), Rest}};
```

Comment:
We will need some access control here.

Either it is the user themselves (if there is an user) or someone with the correct rights on either the module or the returned surveys.